templates/cloud/public-cloud.html

```diff
@@ -92,6 +92,6 @@ <h2 class="p-heading--three">The best management and&nbsp;support</h2>
   </div>
 </section>
 
-{% include "shared/contextual_footers/_contextual_footer.html"  with first_item="_cloud_autopilot" second_item="_cloud_newsletter" third_item="_cloud_further_reading" %}
+{% include "shared/contextual_footers/_contextual_footer.html"  with first_item="" second_item="_cloud_newsletter" third_item="_cloud_further_reading" %}
```

Comment:
could we use _cloud_bootstack for the first item?

Reply:
Done